internal/services/containers/kubernetes_cluster_data_source.go

```diff
@@ -380,6 +380,31 @@ func dataSourceKubernetesCluster() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemOrUserAssignedIdentityComputed(),
 
+			"key_vault_kms": {
+				Type:     pluginsdk.TypeList,
+				Computed: true,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"enabled": {
```

Comment:
It seems like these fields are attribute, not arguments since they don't have `Optional`/`Required` nor `Default`.